schema.sql

```diff
@@ -18,4 +18,45 @@ ALTER TABLE animals DROP COLUMN species;
 ALTER TABLE animals ADD species_id INT;
 ALTER TABLE animals ADD CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id) ON DELETE CASCADE;
 ALTER TABLE animals ADD owner_id INT;
-ALTER TABLE animals ADD CONSTRAINT fk_owners FOREIGN KEY(owner_id) REFERENCES owners(id) ON DELETE CASCADE;
\ No newline at end of file
+ALTER TABLE animals ADD CONSTRAINT fk_owners FOREIGN KEY(owner_id) REFERENCES owners(id) ON DELETE CASCADE;
+
+CREATE TABLE vets(
+	id INT GENERATED BY DEFAULT AS IDENTITY, 
+	name VARCHAR(20), 
+	age INT, 
+	date_of_graduation date, 
+	PRIMARY KEY (id)
+);
+
+CREATE TABLE specializations(
+	species_id INT, 
+	vets_id INT, 
+	CONSTRAINT fk_speciesJoin FOREIGN KEY (species_id) REFERENCES species(id), 
+	CONSTRAINT fk_vetsJoin FOREIGN KEY (vets_id) REFERENCES vets(id)
+);
+
+CREATE TABLE visits(
+	animals_id INT, 
+	vets_id INT, CONSTRAINT fk_animalsJoin FOREIGN KEY (animals_id) REFERENCES animals(id), 
+	CONSTRAINT fk_vetsJoinA FOREIGN KEY (vets_id) REFERENCES vets(id)
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md) for this project, " `There is a many-to-many relationship between the tables animals and vets: an animal can visit multiple vets and one vet can be visited by multiple animals. Create a "join table" called visits to handle this relationship, it should also keep track of the date of the visit.` ", since we have to keep track of the date of the visit, kindly add that field to your `visits` table.

Reply:
@Mihndim2020 Tbh I added the column `date_of_visits` to track the visited date in the **database**, but I forgot to add the command in **schema.sql**. Thanks for pointing that out 🙌.